thunderbird/MozillaThunderbird.spec

```diff
@@ -370,27 +380,19 @@ ac_add_options --disable-elf-hack
 %endif
 ac_add_options --with-system-nspr
 ac_add_options --with-system-nss
-%if %{localize}
-ac_add_options --with-l10n-base=$RPM_BUILD_DIR/l10n
-%endif
-#ac_add_options --with-system-jpeg    # libjpeg-turbo is used internally
-#ac_add_options --with-system-png     # doesn't work because of missing APNG support
 ac_add_options --with-system-zlib
+ac_add_options --with-l10n-base=$RPM_BUILD_DIR/l10n
 ac_add_options --disable-updater
-ac_add_options --disable-tests
-ac_add_options --enable-alsa
-ac_add_options --disable-debug
+#ac_add_options --with-system-png # no apng support
+#ac_add_options --enable-system-hunspell
 ac_add_options --enable-startup-notification
 ac_add_options --disable-necko-wifi
 ac_add_options --enable-update-channel=%{update_channel}
 %if %has_system_cairo
 ac_add_options --enable-system-cairo
 %endif
 ac_add_options --with-unsigned-addon-scopes=app
-%if %branding
 ac_add_options --enable-official-branding
-%endif
```

Comment:
There is no alternative branding AFAIK and there is also no strict trademark policy. So ifdefing in general is not required.

Reply:
Ok